youtube_dl/extractor/rutv.py

```diff
@@ -179,7 +179,7 @@ def _real_extract(self, url):
                         'player_url': 'http://player.rutv.ru/flash3v/osmf.swf?i=22',
                         'rtmp_live': True,
                         'ext': 'flv',
-                        'vbr': int(quality),
+                        'vbr': int(quality) if quality else 0,
```

Comment:
Instead, let's have `int_or_none(quality)`, or if the `quality` string may have punctuation, like '1,000' or '+42', use `str_to_int()`. These functions are in utils.py.

It's better to return a None bitrate than an incorrect number.